pages/components/shared/Spinner.tsx

```diff
@@ -0,0 +1,39 @@
+import Skeleton from "@mui/material/Skeleton";
+import Grid from "@mui/material/Grid";
+import { Box } from "@mui/material";
+
+function TypographyDemo(props: { loading?: boolean }) {
+  const { loading = false } = props;
+
+  return (
+    <Box sx={{ flex: 1 }}>
+      <Box>
+        {loading
+          ? <>
+            <Skeleton sx={{ width: "100%", height: "10vh", margin: 0 }}/> 
+            <Skeleton sx={{ width: "100%", height: "30vh", margin: 0 }}/>
```

Comment:
```suggestion
            <Skeleton sx={{ width: "100%", height: "30vh", margin: 0 }}/>
```

does 30HV makes it dynamic ?